src/Makie.jl

```diff
@@ -297,4 +297,6 @@ export heatmap!, image!, lines!, linesegments!, mesh!, meshscatter!, scatter!, s
 
 export PointLight, EnvironmentLight, AmbientLight, SSAO
 
+@deprecate Node Observable
```

Comment:
This only deprecates `Node` as a function. Is there a way to deprecate it as a type (alias)? I.e. have `foo(x::Node)` work with a deprecation warning?

Reply:
Yeah this will break any type signature using `Node` with a weird error...
I thought about a way to deprecate a type gracefully, but I couldn't think of anything.
I think we should just remove the alias, and have people look at the change log when updating to a major version.

Reply:
We could also show a warning/info message with all breaking changes the first time someone compiles the new Makie version... I somehow don't like that approach, but I also can't find anything bad about it :D

Reply:
`Base.@deprecate_binding`